### PR TITLE
Restructure order

### LINKED ISSUE #7: Reestructurar Tutorial MongoDB
1. 
-    Introduction(Intro)
    -    NoSQL, MongoDB, Database, Collection, Document, keys, JSON/BSON, distribución
-    Conection(C)
    -    Shell, PyMongo

2. 
-    Insertion(I)
-    Simple Selection(F1)
-    Update(U)
-    Delete(D)
3.  
-    Complex Selection(F2)
    -    exact match, elemMatch, array match, geographic math

-    Results Modification
    -    Projection(P)
    -    Sorting(S)
    -    Counting(N)	
-    Indexing(X)
4. 
-    Agregation(A)

### PR DESCRIPTION
**Pull request by pcastelln**
Closes #7

### CODE REVIEW

File: tutorial/00_introduction.ipynb [hunk 1]
```diff
@@ -166,6 +168,21 @@
     "client['admin']"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "aa9b7ed8-0b21-4a2d-b52d-d2c93450b1b4",
+   "metadata": {},
+   "source": [
+    "**`mongosh`:** When using the shell, it is necessary to directly connect to the database of interest. To run `mongosh` inside the docker container, run:\n",
```
Comment (amoya87) [inline tutorial/00_introduction.ipynb]: "**`mongosh`:** To use the shell, you can connect directly to the database of interest by running `mongosh` inside the docker container, run:\n",
Nota: Es posible conectarse al shel de forma remota

File: tutorial/00_introduction.ipynb [hunk 2]
```diff
@@ -11,7 +11,9 @@
...
     "\n",
-    "**Warning:** During this tutorial we'll be connecting to a MongoDB instance with version 6.0 (the latest at time of writing) and using pymongo 4.3.3. As MongoDB is usually updated, there might be some features here that won't work with older versions or that might have somewhat different behaviour depending on the version of the MonogDB instance we'll be connecting to. During this tutorial we'll try and point out the most egregious incompatibilities, but it is not within the scope to handle all of them, so be mindful of the error messages you might receive during regular use. "
+    "The usage of the MongoDB Shell `mongosh` will also be briefly referenced to get the \"standard\" way of using the equivalent pymongo commands. This is generally useful as it is a way of interacting with MongoDB without depending on a specific language. The inputs are the same types as the corresponding pymongo functions unless stated. It is possible to use unquoted strings as keys in the dictionaries in `mongosh`, unless the string contains at least one dot. \n",
```
Comment (amoya87) [inline tutorial/00_introduction.ipynb]: "The usage of the MongoDB Shell `mongosh` will also be briefly referenced. This is generally useful as it is a way of interacting with MongoDB without depending on a specific language. The inputs are the same types as the corresponding pymongo functions unless stated. It is possible to use unquoted strings as keys in the dictionaries in `mongosh`, unless the string contains at least one dot. \n",
Nota:  sugiero cambiar o eliminar la frase (...) to get the \"standard\" way of using the equivalent pymongo commands (...)

File: tutorial/02_indexes_and_advanced_queries.ipynb [hunk 3]
```diff
@@ -2,20 +2,22 @@
...
-    "\n",
-    "For the examples in this tutorial, neither the gain in search speed nor the extra storage required will be noticeable, but for large collections this has to be taken into account.\n",
-    "\n",
-    "Every collection in MongoDB comes indexed by `_id` by default, but new indices can be created at any time and there are different kinds of indices.\n",
+    "## Advanced queries\n",
     "\n",
-    "**Note:** If your user doesn't have write permission, you won't be able to create your own indices, but the gains when doing queries over existing indices will still be noticeable.\n",
+    "In the previous section we saw only the basics of queries. This time we'll be going through some options that can modify the number of documents returned, as well as how to create queries for lists, nested documents and the special case of queries by coordinates.\n",
```
Comment (amoya87) [inline tutorial/02_indexes_and_advanced_queries.ipynb]: "In the previous section we saw only the basics of queries. This time we'll be going through some options that can modify the documents returned, as well as how to create queries for lists, nested documents and the special case of queries by coordinates.\n",
Nota: projection can modify de documents, not only the counts

File: tutorial/02_indexes_and_advanced_queries.ipynb [hunk 4]
```diff
@@ -37,758 +39,908 @@
...
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "6edb3b23-cbc1-4d72-9e34-662ba067b7a5",
+   "id": "25792e27-1cfb-405a-ba4e-cb1ea08bc3bd",
    "metadata": {},
    "source": [
-    "The above creates an index over `lastmjd`, but only for documents with `ndet` greater than 100.\n",
+    "There are a few operators that work only with arrays. We've seen `$in` already, but we'll see a few others now.\n",
```
Comment (amoya87) [inline tutorial/02_indexes_and_advanced_queries.ipynb]: $in operator is not only for arrays. The following query is valid
db.object.find({'aid': {'$in' : ['AL22laceaicnfejpc']}})

File: tutorial/00_introduction.ipynb (hunk 1 above)
Comment (pcastelln) [reply tutorial/00_introduction.ipynb]: Te refieres a cuándo usamos otra herramienta? Acá igual la idea es que, con la forma en que tenemos esto estructurado, lo deberían correr desde el contenedor. Podría añadir una clarificación en alguna parte al respecto, a menos que te refieras a usar otras herramientas

File: tutorial/02_indexes_and_advanced_queries.ipynb (hunk 3 above)
Comment (pcastelln) [reply tutorial/02_indexes_and_advanced_queries.ipynb]: De hecho quité todas las referencias a modificar documentos. Esa sección al final sólo tiene que ver con queries para listas, subdocumentos y geospaciales. Dejaré la nota en la parte de proyecciones

File: tutorial/02_indexes_and_advanced_queries.ipynb (hunk 4 above)
Comment (pcastelln) [reply tutorial/02_indexes_and_advanced_queries.ipynb]: Fixed

File: tutorial/02_indexes_and_advanced_queries.ipynb (hunk 3 above)
Comment (pcastelln) [reply tutorial/02_indexes_and_advanced_queries.ipynb]: Fixed

File: tutorial/00_introduction.ipynb (hunk 2 above)
Comment (pcastelln) [reply tutorial/00_introduction.ipynb]: Fixed

File: tutorial/00_introduction.ipynb (hunk 1 above)
Comment (pcastelln) [reply tutorial/00_introduction.ipynb]: Fixed: Ya está añadida la clarificación

**Review by amoya87 — APPROVED** [review]

**Pull request merged**